src/deploy/hosting/uploader.js

```diff
@@ -179,7 +179,7 @@ class Uploader {
 
   queuePopulate() {
     const pop = this.populateBatch;
-    this.populateQueue.add(pop, "batch" + (this.populateQueue.stats().total + 1));
+    this.populateQueue.add(pop);
```

Comment:
I traced back. This was added in https://github.com/firebase/firebase-tools/commit/484e784777cb2ae770358444fa20292c13e5c4e6#diff-2041a1adf696cab777ab5e6e3670a9e9R176.

`Queue.add()` only takes one argument, and we were ignoring the second argument the whole time.
I discovered this bug because hosting test won't finish. https://travis-ci.org/firebase/firebase-tools/jobs/463079863.
Cheers to whoever added the integration test! It is actually catching some bugs.